src/zope/app/renderer/interfaces.py

```diff
@@ -14,9 +14,8 @@
 """Renderer Interface Declarations
 
 The source renderer takes a special type of string, an ISource, and is able to
-produce 
+produce a different format.
```

Comment:
I love these incomplete documentation sentences.  They raise all kinds of interesting questions!  What happened to the person writing them?